crates/nu-command/src/filesystem/ucp.rs

```diff
@@ -47,6 +47,8 @@ impl Command for UCp {
             )
             .switch("progress", "display a progress bar", Some('p'))
             .switch("no-clobber", "do not overwrite an existing file", Some('n'))
+            .switch("link", "hard-link files instead of copying", Some('l'))
+            .switch("symbolic-link", "make symbolic links instead of copying", Some('s'))
```

Comment:
Flag names and descriptions are copied directly from the uutils.cp docs:
https://uutils.github.io/coreutils/docs/utils/cp.html
```
--link, -l
hard-link files instead of copying

...

--symbolic-link, -s
make symbolic links instead of copying
```